### PR TITLE
my fulton changes

### PR DESCRIPTION
**Pull request by katnep**
fulton rework to make them worth getting
added the medipen from the medevac bundle
made fulton able to carry people
removed medevac because it was made redundant
renamed fulton to "extract and rescue kit" and "extract and rescue pack"

### CODE REVIEW

File: code/modules/mining/machine_vending.dm [hunk 1]
```diff
@@ -160,7 +159,7 @@
 	return ..()
 
 /obj/machinery/mineral/equipment_vendor/proc/RedeemVoucher(obj/item/weapon/mining_voucher/voucher, mob/redeemer)
-	var/items = list("Crusher Kit", "Extraction Kit", "Hunter Kit", "Medivac Kit", "Mining Drone", "Resonator and Advanced Scanner", "Survival Capsule and Explorer's Webbing")//Alphabetical, please.
+	var/items = list("Crusher Kit", "Extract amd Rescue Kit", "Hunter Kit", "Mining Drone", "Resonator and Advanced Scanner", "Survival Capsule and Explorer's Webbing")//Alphabetical, please.
```
Comment (TalkingCactus) [inline code/modules/mining/machine_vending.dm]: `"Extract amd Rescue Kit"` -> `"Extract and Rescue Kit"`

File: code/modules/mining/fulton.dm [hunk 2]
```diff
@@ -10,13 +10,8 @@ var/list/total_extraction_beacons = list()
 	var/list/beacon_networks = list("station")
 	var/uses_left = 3
 	var/can_use_indoors
-	var/safe_for_living_creatures = 0
+	var/safe_for_living_creatures = 1
```
Comment (TalkingCactus) [inline code/modules/mining/fulton.dm:13]: Should probably change the description and name so the players know that they can use this on living things, too.

**Review by TalkingCactus — CHANGES_REQUESTED** [review]
Typo fix as well as a name and description change for the extraction pack.

**Comment by katnep** [comment]
made the changes, had to do it in a really roundabout way but fixed them :p

File: code/modules/mining/fulton.dm [hunk 3]
```diff
@@ -2,21 +2,16 @@ var/list/total_extraction_beacons = list()
 
 /obj/item/weapon/extraction_pack
 	name = "fulton material extraction pack"
```
Comment (TalkingCactus) [inline code/modules/mining/fulton.dm]: `"fulton material extraction pack"` -> `"fulton extraction pack"`

Comment (katnep) [reply code/modules/mining/fulton.dm]: fixed

**Review by TalkingCactus — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request closed without merging**